benchmark/benchmark_driver.py

```diff
@@ -0,0 +1,190 @@
+from datetime import datetime
+import json
+from os import environ, getcwd
+from platform import node, platform
+from sys import stdout
+
+from eliot import start_action
+from eliot.twisted import DeferredContext
+
+from twisted.python.components import proxyForInterface
+from twisted.python.filepath import FilePath
+from twisted.internet.task import cooperate
+from twisted.internet.defer import maybeDeferred, Deferred
+
+from flocker import __version__ as flocker_client_version
+from flocker.apiclient import (
+    IFlockerAPIV1Client, FakeFlockerClient, FlockerClient,
+)
+from flocker.common import gather_deferreds
+
+from benchmark_operations import get_operation
+from benchmark_measurements import get_measurement
+from benchmark_scenarios import get_scenario
+
+
+def benchmark(measure, operation, scenario, n=3):
+    """
+    Perform sampling of the operation.
+
+    :param measure: A quantity to measure.
+    :param operation: An operation to perform.
```

Comment:
Does it really _perform_ the operation? AFAICT, it only calls `get_probe` on it. Maybe I'll understand better as I get further in.